cmd/web-handlers.go

```diff
@@ -293,8 +293,8 @@ objectLoop:
 			return err
 		}
 		if !hasSuffix(object, slashSeparator) {
-			// If not a directory, compress the file and write it to response.
-			err = remove(pathJoin(args.Prefix, object))
```

Comment:
test case was already there, but the browser code also needed. i had assumed rushenn would do multi-delete before a release is made which did not happen. i have made the browser changes now.